taxonomies/minerals.txt

```diff
@@ -150,6 +150,9 @@ wikipedia:en:https://en.wikipedia.org/wiki/Mineral
 vegan:en:yes
 vegetarian:en:yes
 
+en:ammonium bicarbonate
+es:bicarbonato de amonico,bicarbonato de amonio
```

Comment:
isn't it "bicarbonato amonico" (without the "de") ?

Reply:
I've added both - in the ingredients lists it tends to include the "de"